tests/unit/Middleware/Form/FormMiddlewareCest.php

```diff
@@ -229,12 +234,50 @@ public function setCookieAndCsrfOnGetRequest(\UnitTester $I)
             return new Response();
         });
         $prophet->checkPredictions();
-        $flashMessage = SetCookies::fromResponse($response)->get('twigyard_flash_message')->getValue();
-        $flashMessageType = SetCookies::fromResponse($response)->get('twigyard_flash_message_type')->getValue();
-        $csrfToken = SetCookies::fromResponse($response)->get('twigyard_csrf_token')->getValue();
-        $I->assertEquals('token', $csrfToken);
-        $I->assertEquals(null, $flashMessage);
-        $I->assertEquals(null, $flashMessageType);
+        $flashMessageCookie = SetCookies::fromResponse($response)->get('twigyard_flash_message');
+        $flashMessageTypeCookie = SetCookies::fromResponse($response)->get('twigyard_flash_message_type');
+        $csrfTokenCookie = SetCookies::fromResponse($response)->get('twigyard_csrf_token');
+        $I->assertEquals('token', $csrfTokenCookie->getValue());
+        $I->assertEquals(null, $flashMessageCookie->getValue());
+        $I->assertEquals(null, $flashMessageTypeCookie->getValue());
+        $I->assertEquals(false, $csrfTokenCookie->getSecure());
+        $I->assertEquals(false, $flashMessageCookie->getSecure());
+        $I->assertEquals(false, $flashMessageTypeCookie->getSecure());
+    }
+
+    /**
+     * @param \UnitTester $I
+     */
+    public function setCookieAndCsrfOnHttpsGetRequest(\UnitTester $I)
```

Comment:
We do not need test everything again, lets just test the secure flag as that is the only thing that differs from the `setCookieAndCsrfOnGetRequest()`.

Also could we reuse some code between the two methods?